### ISSUE TITLE
Chrome: Beim Ziehen von neuen Elementen aus der linken Liste wird die gesamte Elementliste gezogen 

### ISSUE THREAD

**Issue by jojohoch** [iqb-berlin/verona-modules-aspect#437]

**Comment by rhenck** [comment]
Hierfür finde ich keine Lösung. Hängt irgendwie mit der Kombination Material und native DnD (draggable) zusammen. Internetrecherche ergab nichts dazu.
Workaround über festes DragImage klappt nicht richtig und wäre ohne Funktionsverlust nur aufwändig zu implementieren.
Workround 2 über Benutzung von Material-DnD (statt draggable) funktioniert nicht, weil anscheinend DnD und das ExpansionPanel nicht harmonieren.

**Comment by helfjama** [comment]
Ist das noch aktuell @jojohoch

**Comment by rhenck** [comment]
Ja, ist reproduzierbar. Aber kein Plan, wie man dem beikommen soll.
